css-three-column-layout/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  background-color: #3bb1ff;
+  font-family: Lato, sans-serif;
+  color: white;
+  padding: 50px 100px;
+}
+
+.row {
+  display: flex;
+  justify-content: space-around;
+}
+
+.column-one-third {
+  width: calc(100%/3);
+}
+
+.column-one-third2 {
```

Comment:
you don't want to use the column as the content. you want to use the `column-one-third` class to create the column, and then nest a div in that element and use that as the container for the image with the background color.